src/MasterSite.php

```diff
@@ -507,7 +504,9 @@ function ( $item ) {
 				return ! in_array( 'wpml-ls-item', $item->classes ?? [], true );
 			}
 		);
-		$context['languages']         = $sitepress ? $sitepress->get_ls_languages() : [];
+
+		$context['languages_list'] = function_exists( 'icl_get_languages' ) ? icl_get_languages() : [];
+		$context['languages']      = count( $context['languages_list'] );
```

Comment:
I guess we need to preserve the name `languages` for the count because child themes could override the template? Maybe add a short code comment about this.

Reply:
We can call the new one `site_languages`, as the plural form already implies it's a list.

Reply:
Also a bit easier to read with a `$languages` variable
```
		$languages = function_exists( 'icl_get_languages' ) ? icl_get_languages() : [];

		$context['site_languages'] = $languages;
		$context['languages']      = count( $languages );